### PR TITLE
python: modem-firmware-1.3+: device_credentials_installer

### PR DESCRIPTION
**Pull request by plskeggs**
Add CoAP connectivity and provisioning service CAs as options.

### CODE REVIEW

**Comment by plskeggs** [comment]
What's great about this solution is that we can simply add the CoAP server's root CA cert to slot 0 in addition to the AWS root CA cert, and the device will work with CoAP, MQTT, or REST.

**Comment by plskeggs** [comment]
I should update the readme.

File: python/modem-firmware-1.3+/device_credentials_installer.py [hunk 1]
```diff
@@ -648,6 +656,10 @@ def main():
     print(local_style('Writing AWS CA to modem...'))
     if is_gateway:
         modem_ca = aws_ca.replace("\n", "\\n")
+    elif args.coap:
+        modem_ca = coap_ca + aws_ca
+    elif args.prov:
```
Comment (jayteemo) [inline python/modem-firmware-1.3+/device_credentials_installer.py]: should a warning be printed to let the user know that `--coap` and `--prov` are mutually exclusive?

File: python/modem-firmware-1.3+/README.md [hunk 2]
```diff
@@ -82,6 +83,8 @@ options:
   --port PORT           Specify which serial port to open, otherwise pick from list (default: None)
   --id_str ID_STR       Device ID to use instead of UUID. Will be a prefix if used with --id_imei (default: )
   --id_imei             Use IMEI for device ID instead of UUID. Add a prefix with --id_str (default: False)
+  --coap                Install both the AWS root CA cert as well as the CoAP server root CA cert
```
Comment (jayteemo) [inline python/modem-firmware-1.3+/README.md]:
```suggestion
  --coap                Install the CoAP server root CA cert in addition to the AWS root CA cert
```

File: python/modem-firmware-1.3+/device_credentials_installer.py [hunk 3]
```diff
@@ -110,6 +112,12 @@ def parse_args():
                         Pipe (|) delimited firmware types for FOTA of the set
                         {APP MODEM BOOT SOFTDEVICE BOOTLOADER}; enclose in double quotes
                         """, default="APP|MODEM")
+    parser.add_argument("--coap",
+                        help="Write CA cert for the CoAP Connectivity server, not AWS IoT Core",
```
Comment (jayteemo) [inline python/modem-firmware-1.3+/device_credentials_installer.py]:
```suggestion
                        help="Install the CoAP server root CA cert in addition to the AWS root CA cert",
```

File: python/modem-firmware-1.3+/device_credentials_installer.py [hunk 4]
```diff
@@ -110,6 +112,12 @@ def parse_args():
...
+                        action='store_true', default=False)
+    parser.add_argument("--prov",
+                        help="Write CA cert for CoAP Provisioning server, not AWS IoT Core",
```
Comment (jayteemo) [inline python/modem-firmware-1.3+/device_credentials_installer.py]:
```suggestion
                        help="Install the nrf_provisioning root CA cert",
```

**Review by jayteemo — APPROVED** [review]

**Pull request merged**